cmake/modules/gluecodium/gluecodium/KnownOptionalProperties.cmake

```diff
@@ -254,6 +254,14 @@ _gluecodium_define_target_property(
     "This property is initialized by the value of the GLUECODIUM_IMPORT_FRAMEWORKS_DEFAULT variable if it is set when the function gluecodium_add_generate_command is called."
 )
 
+_gluecodium_define_target_property(
+  GLUECODIUM_SYNCHRONIZE_ACCESS_CLASS_CACHE
+  BRIEF_DOCS "Option to enable synchronization for JNIClassCache hash map."
+  FULL_DOCS
+    "Generates code for synchronized access to the hash map with cached information about JNI classes if this option is set to ON."
```

Comment:
Add also "Such synchronisation might be necessary when Java/JNI generated code is located in several libraries and the libraries are loaded on-demand"